src/client/_clientwindow.py

```diff
@@ -1152,7 +1152,8 @@ def handle_invalid(self, message):
         raise Exception(message)
 
     def handle_stats(self, message):
-        self.statsInfo.emit(message)
+        # self.statsInfo.emit(message)  # currently useless -> safe some time
```

Comment:
Your comment explains the change in https://github.com/FAForever/client/pull/431/commits/0b66c7c15f96b76bf0992a011e251a389bfac3ff#diff-18b50d81a733c795841561309848bd40R202.

But handle_stats is called only, when the server sends stats.